### PR TITLE
fix bug when adding labels to metrics without label filters

### PR DESCRIPTION
**Pull request by arikalon1**

### CODE REVIEW

File: src/robusta/core/sinks/robusta/prometheus_discovery_utils.py [hunk 1]
```diff
@@ -53,11 +53,11 @@ def _get_query_prometheus_value(self, query: str) -> Optional[float]:
...
-            if query_result.result_type == "error" or query_result.vector_result is None:
+            if query_result.result_type == "error" or not query_result.vector_result:
                 logging.error(f"PrometheusDiscoveryUtils failed to get prometheus results.")
                 return
             value = query_result.vector_result[0].value.value
-            return_value = float('%.2f' % float(value))
+            return_value = float("%.2f" % float(value))
             return return_value if return_value >= 0 else None
         except:
             logging.exception(f"PrometheusDiscoveryUtils failed to get prometheus results.")
```
Comment (itisallgood) [inline src/robusta/core/sinks/robusta/prometheus_discovery_utils.py:63]: f-string here is redundant

File: src/robusta/core/sinks/robusta/prometheus_discovery_utils.py [hunk 2]
```diff
@@ -53,11 +53,11 @@ def _get_query_prometheus_value(self, query: str) -> Optional[float]:
             global_config = self.__global_config
             prometheus_params = PrometheusParams(**global_config)
             query_result = run_prometheus_query(prometheus_params=prometheus_params, query=query)
-            if query_result.result_type == "error" or query_result.vector_result is None:
+            if query_result.result_type == "error" or not query_result.vector_result:
                 logging.error(f"PrometheusDiscoveryUtils failed to get prometheus results.")
```
Comment (itisallgood) [inline src/robusta/core/sinks/robusta/prometheus_discovery_utils.py:57]: f-string in this log.exception is redundant as well

File: src/robusta/core/sinks/robusta/prometheus_discovery_utils.py (hunk 1 above)
Comment (arikalon1) [reply src/robusta/core/sinks/robusta/prometheus_discovery_utils.py]: fixed

File: src/robusta/core/sinks/robusta/prometheus_discovery_utils.py (hunk 2 above)
Comment (arikalon1) [reply src/robusta/core/sinks/robusta/prometheus_discovery_utils.py]: fixed

**Pull request merged**